specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/examples/2018-01-01/sms_received.json

```diff
@@ -9,7 +9,8 @@
     "From": "15555555555",
     "To": "15555555555",
     "Message": "Great to connect with ACS events",
-    "ReceivedTimestamp": "2020-09-18T00:27:45.32Z"
+    "ReceivedTimestamp": "2020-09-18T00:27:45.32Z",
```

Comment:
nit: can we align the example here with the properties above --- is it `ReceivedTimestamp` and `SegmentCount` or `receivedTimestamp` and `segementCount` ?